packages/containers/src/List/selector.js

```diff
@@ -65,7 +65,7 @@ export function configureGetFilteredItems(configure) {
 
 export function compare(sortBy) {
 	return (a, b) => {
-		if (a.get(sortBy)) {
+		if (a.get(sortBy) !== undefined) {
```

Comment:
You'll have the same issue with null. In fact the comparison in this block seems to work only for string and number. What about testing that the value is a string or a number ?